package.json

```diff
@@ -48,15 +48,16 @@
     "eslint-plugin-import": "^2.11.0",
     "mocha": "^5.1.1",
     "nyc": "^14.1.0",
+    "rimraf": "^3.0.0",
```

Comment:
@profnandaa [I suggested adding rimraf](https://github.com/validatorjs/validator.js/pull/1112#discussion_r334507426) because ```rm -rf``` is UNIX specific. In the initial version of this PR, @CodeIter added some windows specific cleaning commands that can be merged with the existing command if we use rimraf.

Reply:
@tux-tn -- I see, thanks for the clarification.